src/components/portfolio/PortfolioPreview.vue

```diff
@@ -1,11 +1,17 @@
 <script setup lang="ts">
 import { Category } from '@enums/category.enum'
-import { getPreviewImageUrl } from '@utils/imageUtils'
+import { getImageUrl } from '@utils/imageUtils'
 </script>
 
 <template>
-  <li>
-    <img :src="getPreviewImageUrl(category, imageSrc)" :alt="imageDesc" class="static-image" />
+  <li style="margin: 8px">
+    <a class="hidden" :href="target">
+      <img
+        :src="getImageUrl(category, `/${target}/${imageSrc}`)"
```

Comment:
This is a perfect opportunity to use a computed property instead of inline concat!